docs/modules/opa/pages/usage-guide/resources.adoc

```diff
@@ -2,13 +2,14 @@
 
 include::concepts:stackable_resource_requests.adoc[]
 
-A minimal OPA setup consists of 1 Pod per Node (DaemonSet) and has the following https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/[resource requirements] per scheduled Pod:
+A minimal OPA setup consists of 1 Pod per node (DaemonSet) and has the following https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/[resource requirements] per scheduled Pod:
```

Comment:
Node could also be seen as Kubernetes term ;)

Reply:
Oh, I see! https://kubernetes.io/docs/concepts/architecture/nodes/

Reply:
it is capitalized when talking about the k8s Ojbect, and lowercased otherwise. So there is a bit of a mix now, but I adjusted some spellings: https://github.com/stackabletech/opa-operator/pull/584/commits/3a3e1f4b0db1084d69a906912f1e2cb523d7dae3

Reply:
Thanks!